### PR TITLE
Add explanation on how to use xdebug

### PR DESCRIPTION
**Pull request by t2oh4e**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,5 +163,11 @@ and needs to be adjusted when using a lower version.
 - php81
 - php82
 
+## PHP debugging
+As mentioned in the features section XDebug is shipped with devenv, but you need to activate it to actully debug something. For the Frontend you can use the [XDebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?gl=US&hl=en) extension when using Chrome. Alternatively you can simply add a query parameter to the URL you want to debug like this: `https://127.0.0.1:8000/?XDEBUG_SESSION=1`
```
Comment (sobyte) [inline README.md:167]:
```suggestion
As mentioned in the features section, XDebug is shipped with devenv and you need to activate it to actually debug something. For the Frontend you can use the [XDebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?gl=US&hl=en) extension when using Chrome. Alternatively, you can simply add a query parameter to the URL you want to debug like this: `https://127.0.0.1:8000/?XDEBUG_SESSION=1`
```

File: README.md [hunk 2]
```diff
@@ -163,5 +163,11 @@ and needs to be adjusted when using a lower version.
...
+This also works great to debug API request in Insomnia or something similar.
```
Comment (sobyte) [inline README.md:168]:
```suggestion
This also works great when having to debug API reques, e.g in Insomnia.
```

File: README.md [hunk 3]
```diff
@@ -163,5 +163,11 @@ and needs to be adjusted when using a lower version.
...
+
+For the CLI you can use an integrated debug helper by simply adding `debug` before your CLI commands, e.g. `debug bin/console`.
```
Comment (sobyte) [inline README.md:170]:
```suggestion
In the CLI you can use an integrated debug helper by simply adding `debug` before your CLI commands, e.g. `debug bin/console`.
```

File: README.md (hunk 2 above)
Comment (Wysselbie) [inline README.md:168]: [Xdebug](https://xdebug.org/) itself uses "Xdebug".
```suggestion
As mentioned in the features section Xdebug is shipped with devenv, but you need to activate it to actually debug something. For the Frontend you can use the [Xdebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?gl=US&hl=en) extension when using Chrome. Alternatively you can simply add a query parameter to the URL you want to debug. Like this: `https://127.0.0.1:8000/?XDEBUG_SESSION=1`
This also works great to debug API requests in Insomnia or something similar.
```

File: README.md (hunk 3 above)
Comment (Wysselbie) [inline README.md:170]:
```suggestion
For the CLI you can use the integrated debug helper `debug` by simply adding it before your CLI commands, e.g. `debug bin/console`.
```

**Review by Wysselbie — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Wysselbie) [reply README.md:168]: I like this more then my suggestion to line 168. 👍

File: README.md (hunk 1 above)
Comment (t2oh4e) [reply README.md]: Thank you! Applied.

File: README.md (hunk 2 above)
Comment (t2oh4e) [reply README.md]: Thank you! Applied.

File: README.md (hunk 3 above)
Comment (t2oh4e) [reply README.md]: Thank you! Applied.

**Review by Wysselbie — APPROVED** [review]
Looks good to me.
Nice, that you've corrected the "Xdebug" above. 👍

**Pull request merged**